### PR TITLE
Marlon/99/start game

### PR DESCRIPTION
**Pull request by marlonanthony**
Here's a random hampster🐹

### CODE REVIEW

File: WheelOfFortune/Game.cs [hunk 1]
```diff
@@ -50,43 +50,29 @@ public class Game
...
-            Console.ReadKey(true);
-            AddPlayer();
+            Prompt.WelcomeMessage();
 
-            Random random = new();
-            int index = random.Next(0, allPuzzles.Count);
-            string nextPuzzle = allPuzzles[index];
-            Puzzle puzzle = new(nextPuzzle);
-            CurrentPuzzle = puzzle;
+            string person = Prompt.CreatePlayer();
```
Comment (cychoi00) [inline WheelOfFortune/Game.cs:55]: Thinking ahead to potential feedback we might receive - would it maybe be better for this variable to be named `name` or `playerName`?

File: WheelOfFortune/Game.cs [hunk 2]
```diff
@@ -50,43 +50,29 @@ public class Game
...
+            Player player = new Player(person);
```
Comment (cychoi00) [inline WheelOfFortune/Game.cs:56]: Same here - would it be better for the variable to be something like `newPlayer`?

File: WheelOfFortune/Game.cs [hunk 3]
```diff
@@ -50,43 +50,29 @@ public class Game
...
-            while (string.IsNullOrEmpty(name))
+            char addNewPlayerResponse = Prompt.AddAdditionalPlayer();
+            while (addNewPlayerResponse == 'y')
             {
-                Console.WriteLine("Vanna White would like to know your name. Please input your name once more so we can play the game!");
-                name = Console.ReadLine();
+                person = Prompt.CreatePlayer();
+                player = new Player(person);
+                Players.Enqueue(player);
+                addNewPlayerResponse = Prompt.AddAdditionalPlayer();
```
Comment (cychoi00) [inline WheelOfFortune/Game.cs:65]: A stylistic comment (and I'm not sure if one is better than the other) - alternatively, the while loop condition could also be written as `while (Prompt.AddAdditionalPlayer() == 'y')`, allowing you to get rid of lines 59 and 65. But I also think that having variable names can sometimes add to the readability of the code, so up to you on what you think is better!

File: WheelOfFortune/Game.cs (hunk 1 above)
Comment (marlonanthony) [reply WheelOfFortune/Game.cs:55]: Agreed

File: WheelOfFortune/Game.cs (hunk 3 above)
Comment (cychoi00) [reply WheelOfFortune/Game.cs]: Also stylistic - you could combine the first call for the player and additional players by using a `do...while` loop I believe

File: WheelOfFortune/Game.cs (hunk 2 above)
Comment (marlonanthony) [reply WheelOfFortune/Game.cs]: Changed

File: WheelOfFortune/Game.cs (hunk 3 above)
Comment (cychoi00) [reply WheelOfFortune/Game.cs]: If you decide to update the variable names above - you might also want to change these accordingly!

**Review by cychoi00 — APPROVED** [review]
Added some comments on code style that you can take or leave! :shipit:

File: WheelOfFortune/Game.cs (hunk 3 above)
Comment (marlonanthony) [reply WheelOfFortune/Game.cs]: I like both ideas. Give me a moment to make those changes

**Pull request merged**